.env.example

```diff
@@ -8,15 +8,23 @@
 # Your local .env file is added to .gitignore and
 # wont be uploaded to the repo.
 
+SECRET_TOKEN=change_me
```

Comment:
are we using this? I mean in production we populate it during provisioning, right?